src/components/LandingPage/AboutEventTile.tsx

```diff
@@ -0,0 +1,87 @@
+import Image from "next/image";
+import type { ReactNode } from "react";
+
+const eventDetailsSectionStyles = "flex flex-col items-center bg-white";
```

Comment:
could you change the all the variables that have styles to be `UPPERCASE_SNAKECASE`? just so we can differentiate between variables and functions for readability